### ISSUE TITLE
Bug: Reporter doesn't work for non BDD setup projects

### ISSUE THREAD

**Issue by laytong** [vitalets/playwright-bdd#143]
**Given**
```js
import 'dotenv/config';
import { defineConfig, devices } from '@playwright/test';
import { defineBddConfig, cucumberReporter } from 'playwright-bdd';

const testDir = defineBddConfig({
  paths: ['tests/features/*'],
  import: ['tests/steps/**/*.ts', '!tests/fixtures/*'],
  importTestFrom: 'tests/steps/fixtures.ts',
});

export default defineConfig({
  testDir,
  reporter: [
    ['line'],
    cucumberReporter('junit', {
      outputFile: 'coverage/junit.xml',
    }),
    cucumberReporter('html', {
      outputFile: 'coverage/web.html',
    }),
  ],
  retries: 2,
  projects: [
    {
      name: 'setup',
      testMatch: /setup\.ts/,
      testDir: 'auth',
    },
    {
      name: 'chromium',
      use: {
        ...devices['Desktop Chrome'],
      },
      dependencies: ['setup'],
    },
});

```
**When**
When I run the "setup" job and there's an error

**Then**
The reporter falls over for non BDD driven projects and you can't opt out of cucumber report for that project
```
Error in reporter Error: __bddData annotation is not found for test "reservation user".
    at TestCaseRun.extractBddData (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright-bdd/src/reporter/cucumber/messagesBuilder/TestCaseRun.ts:77:13)
    at new TestCaseRun (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright-bdd/src/reporter/cucumber/messagesBuilder/TestCaseRun.ts:53:25)
    at MessagesBuilder.onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright-bdd/src/reporter/cucumber/messagesBuilder/Builder.ts:58:25)
    at Object.onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright-bdd/src/reporter/cucumber/messagesBuilder/index.ts:21:34)
    at CucumberReporterAdapter.onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright-bdd/src/reporter/cucumber/index.ts:56:29)
    at ReporterV2Wrapper.onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/reporters/reporterV2.js:87:150)
    at /Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/reporters/multiplexer.js:47:65
    at wrap (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/reporters/multiplexer.js:86:5)
    at Multiplexer.onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/reporters/multiplexer.js:47:45)
    at InternalReporter.onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/reporters/internalReporter.js:62:20)
    at JobDispatcher._reportTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/runner/dispatcher.js:519:20)
    at JobDispatcher._onTestEnd (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/runner/dispatcher.js:285:10)
    at WorkerHost.emit (node:events:514:28)
    at ChildProcess.<anonymous> (/Users/lgilbraith/code/OrderIntegrity/node_modules/playwright/lib/runner/processHost.js:105:16)
    at ChildProcess.emit (node:events:514:28)
    at emit (node:internal/child_process:951:14)
```
**But I expect**
To be able to run without error

**Isolated demo**
I'll try to create a repro soon

**Environment**
```
Playwright-bdd environment info:

platform: darwin
node: v20.10.0
playwright-bdd: v6.2.0
@playwright/test: v1.43.1
@cucumber/cucumber: v10.5.1
Playwright config file: playwright.config.ts

```

**Comment by vitalets** [comment]
Released in **v6.3.1**.
Please re-check on your side.

**Comment by laytong** [comment]
Thanks, I will take a look!

**Comment by vitalets** [comment]
Looks like the same as https://github.com/vitalets/playwright-bdd/issues/166
Fixed in **v6.6.0**.
@magarwal19 could you re-check on your side?

**Comment by magarwal19** [comment]
Hi It seems to be working as expected currently . We can close this issues.. and if observed again in future will log a new with more details

**Comment by vitalets** [comment]
Ok! Thanks for the feedback!

**Comment by aluckin3** [comment]
@vitalets i see that issue again:
`Error in reporter Error: __bddData annotation is not found for test "Client Portal - Test notification (email)".
at TestCaseRun.extractBddData (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright-bdd\src\reporter\cucumber\messagesBuilder\TestCaseRun.ts:76:13)
at new TestCaseRun (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright-bdd\src\reporter\cucumber\messagesBuilder\TestCaseRun.ts:52:25)
at MessagesBuilder.onTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright-bdd\src\reporter\cucumber\messagesBuilder\index.ts:68:25)
at Object.onTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright-bdd\src\reporter\cucumber\messagesBuilder\ref.ts:24:34)
at CucumberReporterAdapter.onTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright-bdd\src\reporter\cucumber\index.ts:61:29)
at ReporterV2Wrapper.onTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\reporters\reporterV2.js:87:150)
at C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\reporters\multiplexer.js:47:65
at wrap (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\reporters\multiplexer.js:86:5)
at Multiplexer.onTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\reporters\multiplexer.js:47:45)
at InternalReporter.onTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\reporters\internalReporter.js:62:20)
at JobDispatcher._reportTestEnd (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\runner\dispatcher.js:518:20)
at JobDispatcher._failTestWithErrors (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\runner\dispatcher.js:370:10)
at JobDispatcher._massSkipTestsFromRemaining (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\runner\dispatcher.js:377:14)
at JobDispatcher._onDone (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\runner\dispatcher.js:443:10)
at WorkerHost.emit (node:events:518:28)
at ChildProcess.<anonymous> (C:\BuildAgent\work\a556589622e3634b\e2e-tests\node_modules\playwright\lib\runner\processHost.js:105:16)
at ChildProcess.emit (node:events:518:28)
at emit (node:internal/child_process:951:14)
at processTicksAndRejections (node:internal/process/task_queues:83:21)`

It is only present with file that has "@mode:serial" added.

**Comment by vitalets** [comment]
I will check it, reopenning.

**Comment by vitalets** [comment]
Fixed in **v7.3.0**.
@aluckin3 could you check on your side?
